app/views/vacancies/index.html.haml

```diff
@@ -41,7 +41,10 @@
               .govuk-heading-m= t('jobs.no_jobs.radius_suggestions.heading.no_keyword')
             %ul.govuk-list.govuk-list--bullet
               - @vacancies_search.wider_radius_suggestions.each do |wider_radius|
-                %li= link_to("#{wider_radius.first} miles", jobs_path(@vacancies_search.params_hash.merge(radius: wider_radius.first)), class: "govuk-link wider-radius-gtm") + ' ' + t('jobs.no_jobs.radius_suggestions.results', count: wider_radius.last)
+                - if @vacancies_search.location_search.location_polygon.present?
+                  %li= link_to("#{wider_radius.first} miles", jobs_path(@vacancies_search.params_hash.merge(buffer_radius: wider_radius.first)), class: "govuk-link wider-radius-gtm") + ' ' + t('jobs.no_jobs.radius_suggestions.results', count: wider_radius.last)
```

Comment:
would be slightly better to pass `wider_radius.first` into a template in yml like `radius_link: {radius} miles`